.eslintrc.js

```diff
@@ -14,5 +14,8 @@ module.exports = {
 
```

Comment:
Going to leave as-is for now unless you feel strongly. My next PR will allow the user to toggle-off the extension entirely (so the tab won't even show up).